decentraland/unity-renderer
# Feat: added nft carousel for new passport

## What does this PR change?

Fix #3328
Adds new nft carousel component for new passport

...

## How to test the changes?

Cannot really be tested, verify the code
1. Go to: https://play.decentraland.zone/?renderer-branch=feat/nft-carousel
3. Verify that the carousel works fine where previously used (like in the exploreV2 menu)


## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
